doc: Add the downlink path unavailable error with multicast devices (#1328)
TheThingsIndustries/lorawan-stack-docs: doc/content/devices/troubleshooting/_index.md

The `COLLISION_PACKET` error occurs when downlink transmissions overlap - this happens when two or more packets overlap in time and use the same spreading factor, bandwith and frequency plan settings.

To avoid packet collisions, users can enable server-side buffering of donwlink messages. If server-side buffering is enabled, the Gateway Server schedules the downlink message to be sent after the downlink message that was already queued but not sent yet, so that their transmissions don't overlap. To enable this, navigate to your gateway's **General settings** section, expand the **LoRaWAN** section and **Enable** the **Schedule downlink late** option by ticking the box. Note that this is a recommended setting for gateways that use [UDP packet forwarder]({{< ref "/gateways/concepts/udp" >}}).

> ## `no downlink path available` error in device live events while scheduling downlinks for a Multicast device.

Reviewer: ```suggestion
## Scheduling downlinks for a Multicast device results with "no downlink path available" error.
```

> [Multicast]({{< ref "/devices/configuring-devices/multicast" >}}) devices in {{% tts %}} do not support uplinks, confirm downlinks, or MAC commands, so a specified downlink path is required for scheduling multicast downlink messages. Omitting the gateway ID or selecting an inactive gateway in the downlink message causes a `No downlink path available` error. 

Reviewer: ```suggestion
[Multicast]({{< ref "/devices/configuring-devices/multicast" >}}) devices in {{% tts %}} do not support uplinks, confirm downlinks, or MAC commands, so a specified downlink path is required for scheduling multicast downlink messages. For example, omitting the gateway ID or selecting an inactive gateway in the downlink message causes a `No downlink path available` error. 
```

> Below is an example cURL command for scheduling a multicast device downlink.

Reviewer: ```suggestion
Below is an example cURL command for scheduling a downlink to a multicast device:
```
